src/spyglass/spikesorting/v1/sorting.py

```diff
@@ -241,19 +241,41 @@ def make(self, key: dict):
         else:
             # Specify tempdir (expected by some sorters like mountainsort4)
             sorter_temp_dir = tempfile.TemporaryDirectory(dir=temp_dir)
+        
+
             sorter_params["tempdir"] = sorter_temp_dir.name
+
+            if sorter == 'mountainsort5':
+                 sorter_params.pop("tempdir", None)
+            
+            
             # if whitening is specified in sorter params, apply whitening separately
             # prior to sorting and turn off "sorter whitening"
-            if sorter_params["whiten"]:
+            if sorter_params.get("whiten", False):
                 recording = sip.whiten(recording, dtype=np.float64)
                 sorter_params["whiten"] = False
-            sorting = sis.run_sorter(
-                sorter,
-                recording,
-                output_folder=sorter_temp_dir.name,
-                remove_existing_folder=True,
-                **sorter_params,
-            )
+
+            if 'kilosort' in sorter:
```

Comment:
By moving these out of the if clause, we highlight what's unique about the kilosort 'if'
```suggestion

            common_sorter_items = {
                    sorter_name=sorter,
                    recording=recording,
                    output_folder=sorter_temp_dir.name,
                    remove_existing_folder=True
                }
            
            if 'kilosort' in sorter:
```